src/graphs/default.graphml

```diff
@@ -6,7 +6,7 @@
   <graph edgedefault="directed">
     <node id="0">
         <data key="version">25.0</data>
-        <data key="bitcoin_config">uacomment=w0</data>
+        <data key="bitcoin_config">uacomment=w0, debug=addrman</data>
```

Comment:
whats this for?

Reply:
Just to show how it's done, now that I set debug=0 as default